ansible_ai_connect/ai/api/model_pipelines/config_loader.py

```diff
@@ -28,41 +28,48 @@
 
 
 def load_config() -> Configuration:
+    return load_config_from_str(settings.ANSIBLE_AI_MODEL_MESH_CONFIG)
+
+
+def load_config_from_str(model_pipeline_config: str) -> Configuration:
     # yaml.safe_load(..) seems to also support loading JSON. Nice.
     # However, try to load JSON with the correct _loader_ first in case of corner cases
+    result = {}
     errors: [Exception] = []
-    result = load_json()
-    if isinstance(result, Exception):
-        errors.append(result)
-        result = load_yaml()
+
+    if model_pipeline_config != "":
```

Comment:
you can just do `if model_pipeline_config:` Since `""` is considered `False` and this allow you to handle the case value is `0`, `[]`, `{}` or `None`, all of them being cases you want to avoid..

Reply:
Added as new commit @goneri 

I'll squash the commits when merging.